tests/test_lib_caconnector.py

```diff
@@ -555,6 +557,54 @@ def test_04_test_pending_request(self):
             r = cacon.get_issued_certificate(request_id)
             self.assertTrue(r.startswith("-----BEGIN CERTIFICATE-----"), r)
 
+    def test_10_fail_ca_missing_certs(self):
```

Comment:
You can add the `@unittest.skipUnless` decorator to the `MSCATestCase` class so you don't have to add it to every function.

Reply:
Looks to me as if this is not needed for `test_10...`.